ibm_catalog.json

```diff
@@ -0,0 +1,192 @@
+                "slz",
+                "IaC",
+                "infrastructure as code",
+                "terraform",
+                "solution",
+                "VPN",
+                "OpenVPN"
+            ],
+            "short_description": "Creates client-to-site VPN connectivity to landing zones using VPN Server ",
+            "long_description": "By default, the landing zone patterns are configured with private networks not available over the internet.  To access these networks, there are several connectivity options.  This deployable architecture pattern configures the client-to-site VPN Server connectivity with only a few required inputs to configure it within an existing landing zone.  Once deployed, you can install an OpenVPN client application and import a profile from the VPN Server on the devices you want to access the VPN.  The configuration can include a list of users that will be provided access to the private network, controlled by IBM Cloud IAM.\n",
+            "offering_docs_url": "https://ibm.ent.box.com/notes/1212695139323",
+            "offering_icon_url": "https://broker-staging.vpnaasdev.cloud.ibm.com/C2S_icon.svg",
+            "features": [
+                {
+                    "description": "A private certificate engine is configured in the Secret Manager instance. A secret group is created. A private certificate (the \"secret\") using the prvate certificate engine is created in the secret group created above\n",
+                    "title": "Creates and configures an optional Secrets Manager instance, or uses existing."
+                },
+                {
+                    "description": "The network ACL on this subnet grants all access from any source.\n",
+                    "title": "A subnet named 'client-to-site-subnet' in the landing-zone management VPC"
+                },
+                {
+                    "description": "A new security group named 'client-to-site-sg' that allows all incoming request from any source\n",
+                    "title": "client-to-site Security Group"
+                },
+                {
+                    "description": "An IAM access group allowing users to authenticate and connect to the client-to-site VPN gateway\n",
+                    "title": "A new IAM Access Group for VPN users"
+                },
+                {
+                    "description": "Using the private certificate generated above and stored in the Secret Manager instance, the gateway is located in the 'client-to-site-subnet' subnet mentioned, and attaches the 'client-to-site-sg' to the client-to-site VPN gateway with routes configured to allow accessing the landing zone VPCs (management and workload)\n",
+                    "title": "A client-to-site VPN server"
+                }
+            ],
+            "flavors": [
+                {
+                    "label": "Standard",
+                    "name": "standard",
+                    "install_type": "extension",
```

Comment:
I think we'll need the working directory here
` "working_directory": "examples/landing-zone"`

Reply:
Didn't we say not to use `examples` directory for DAs due to the fact we say in our [docs](https://terraform-ibm-modules.github.io/documentation/#/versioning) "changes to examples do not drive release levels" which would not be true for DAs

The PowerVS guys are using [solutions](https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/tree/main/solutions). In `terraform-ibm-iam-account-settings` I used [ibm-catalog](https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/tree/main/solutions/ibm-catalog).